### PR TITLE
Removes the c= parameter and 'tlsa' as an option.

### LINKED ISSUE #12: Resolve c=tlsa option
From the FAQ, This constraint mostly produces confusion, since a policy with "report-only mode" suggests that DANE TLSA should not be enforced (which is contrary to the DANE protocol!). So we can assume we only ever want to send reports for DANE failures; we don't want to change DANE behavior. Thus if we simply specify that STS reports may or should contain statistics around additional authentication failures--including any from future protocols not yet designed--we are both future-proof and able to remove c=tlsa.

This is mostly a no-op; we should change reporting thus anyway, so it really just clarifies the draft.

### PR DESCRIPTION
**Pull request by mrisher**
Fixes #12.

### CODE REVIEW

**Comment by mrisher** [comment]
I think this one was non-controversial, but just leaving it as a Pull Req. so we can discuss en masse.

I probably shouldn't have mixed the new version number changes into this. I'll put those in their own commit as well.

/m

File: spec.md [hunk 1]
```diff
@@ -3,13 +3,13 @@
    Title = "SMTP Strict Transport Security"
    abbrev = "SMTP-STS"
    category = "std"
-   docName = "draft-margolis-smtp-sts-00"
+   docName = "draft-ietf-uta-mta-sts-00"
```
Comment (danmarg) [inline spec.md]: I would normally make that change in a different commit, but whatever. ;)

File: spec.md [hunk 2]
```diff
@@ -371,26 +362,12 @@ authenticated before use.
 ## Policy Validation
 
 When sending to an MX at a domain for which the sender has a valid and
-non-expired SMTP STS policy, a sending MTA honoring SMTP STS SHOULD validate
+non-expired SMTP STS policy, a sending MTA honoring SMTP STS MUST validate
```
Comment (danmarg) [inline spec.md]: Do we want to call out the possibility for a sending MTA to be non-STS-aware but to still do offline reporting? Or is that not necessary?

Comment (mrisher) [inline spec.md]: Since we're planning to move reporting to its own thing, I don't feel it's
necessary here.

/m

**Comment by prbinu** [comment]
@mrisher  if this is complete, please merge it

**Comment by mrisher** [comment]
I think this is already in done. So abandoning.
